src/screens/WalletScreen/Content/Content.tsx

```diff
@@ -18,29 +20,70 @@ export const Content: React.FC<ContentProps> = ({
 	setAddFundsVisible,
 	setSendModalOpen
 }) => {
+	const [mined, setMined] = useState(false);
 	const { i18n } = useLanguage();
 	const navigation = useNavigation();
-	const { loading, fetchTransactions } = useTransactions();
+	const { loading, fetchTransactions, pendingTransactions, setPendingTransactions } = useTransactions();
+
+	useEffect(() => {
+		const fetchStatus = async () => {
+			if (pendingTransactions[0]?.hash) {
+				const provider = await getProvider();
+				await provider.waitForTransaction(pendingTransactions[0]?.hash);
+				setMined(true);
+			} else {
+				setMined(false);
+			}
+		};
+
+		fetchStatus();
+	}, [pendingTransactions[0]?.hash]);
+
+	const handleRefresh = useCallback(() => {
+		fetchTransactions();
+		if (mined) setPendingTransactions([]);
+	}, [fetchTransactions]);
```

Comment:
this should not get only the first pending transaction. one idea:

1) fetchStatus could go through all the pending transactions and if it's mined it could set pending = false, success = true / false in the transaction. It will still show but not as pending.